banksystem/accounts/urls.py

```diff
@@ -1,8 +1,7 @@
 from django.urls import path
-
-from . import views
+from .views import accounts
 
 
 urlpatterns = [
-    path('', views.index, name='index'),
+    path('<str:bank_name>/', accounts, name='accounts'),
```

Comment:
we should always try to use id in urls
`    path('<int:bank_id>/', accounts, name='accounts'),`